render/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/render/jpql/introspector/JpqlRenderIntrospector.kt

```diff
@@ -15,7 +16,8 @@ class JpqlRenderIntrospector(
 ) : AbstractRenderContextElement(Key) {
     companion object Key : RenderContext.Key<JpqlRenderIntrospector>
 
-    private val tableLookupCache: MutableMap<KClass<*>, JpqlEntityDescription> = ConcurrentHashMap()
+    private val classTableLookupCache: MutableMap<KClass<*>, JpqlEntityDescription> = ConcurrentHashMap()
+    private val propertyTableLookupCache: MutableMap<KCallable<*>, JpqlPropertyDescription> = ConcurrentHashMap()
```

Comment:
The `table` prefix is a misnomer on my part - I named it table because I was planning to have native queries at that time, but now that it's JPQL-only, I think it's okay to name it `entityLookupCache` and `propertyLookupCache`.